SpatialGDK/Source/Public/SpatialConstants.h

```diff
@@ -6,8 +6,6 @@
 
 #include <improbable/c_worker.h>
 
-using Worker_EntityId_Key = int64;
```

Comment:
We need this for linux, potentially add a comment for this so that we don't attempt this again

Reply:
Could you explain why this breaks on Linux - just so I can add some context for the comment.